src/app/action-items/action-items.component.ts

```diff
@@ -20,10 +26,12 @@ export class ActionItemsComponent implements OnInit, OnDestroy {
   dataSource: ActionItemMapped[];
   isLoadingActionItems = false;
   isLoadingActionItem = false;
-  actionItemId: string | undefined;
-  actionItem: ActionItem;
-  errorMessage: string | undefined;
-  justAddedItemId: string;
+  actionItemId?: string;
```

Comment:
**CONCERN:** Do we need both this variable? Can't we use `actionItem?.id` instead?